Jenkinsfile

```diff
@@ -146,6 +153,8 @@ def job = {
         return null
     }
 
+    // Build on-prem ksqlDB image
```

Comment:
is this comment still relevant since we don't build cloud images here?

Reply:
Nope, removed.